unique_toolkit/unique_toolkit/chat/service.py

```diff
@@ -25,21 +27,19 @@ class ChatService:
     Provides all functionalities to manage the chat session.
 
     Attributes:
-        event (Event): The Event object.
+        event (Event | ChatEvent): The Event object.
         logger (Optional[logging.Logger]): The logger. Defaults to None.
     """
 
     def __init__(self, event: ChatEvent | Event):
+        self.event = event
```

Comment:
Why do we need the event, since basically most of what we use in the event are already added to the class properties in the subsequent lines...